### PR TITLE
Generate learning path markdown

### PR DESCRIPTION
**Pull request by tsadler1988**
Adds a script for generating markdown files required for hosting Learning Path on innersourcecommons.org.

Only supports English articles initially - intent is to extend for non-English as part of #249.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -10,4 +10,14 @@ node substitute_article_urls.js $TARGET
...
+```
+
+## generate_learning_path_markdown.js
+
+A node script to generate markdown files required for hosting Learning Path on innersourcecommons.org.
+
+### Usage:
+```
+npm ci
+node generate_learning_path_markdown.js
```
Comment (lenucksi) [inline scripts/README.md:22]: Do we need to integrate this into some sort of GitHub Action?

Comment (tsadler1988) [reply scripts/README.md:22]: That would be the long term aim I think yes. 

In my opinion, we should have a GitHub Action generate the markdown and raise a Pull Request on https://github.com/InnerSourceCommons/innersourcecommons.org if it had changed.

**Comment by claredillon** [comment]
From @lenucksi : Waiting to next week for input from Tom or Russ. Be good to have some desired use documented. This looks like it is related to https://github.com/InnerSourceCommons/innersourcecommons.org/pull/189

**Comment by tsadler1988** [comment]
> Thanks, Tom. This works, and I also wanted to suggest [this improvement](https://github.com/tsadler1988/InnerSourceLearningPath/pull/1/files) that makes it a bit easier to use.

Thanks Russ, cherry picked your commit into this PR.

**Pull request merged**